sfa_dash/blueprints/base.py

```diff
@@ -185,6 +185,19 @@ def get_breadcrumb_dict(self):
         """
         return OrderedDict()
 
+    def flash_api_errors(self, errors):
+        """Formats a dictionary of api errors and flashes them to the user on
+        the next request.
+
+        Parameters
+        ----------
+        errors: dict
+            Dict of errors returned by the API.
+        """
+        to_flash = [f'({key}) msg.join(", ")' for key, msg in errors]
```

Comment:
```suggestion
        to_flash = [f'({key}) {msg.join(", ")}' for key, msg in errors]
```
and do we need errors.items(). make sure to test this